src/containerapp/azext_containerapp/_params.py

```diff
@@ -337,11 +337,9 @@ def load_arguments(self, _):
         c.ignore('disable_max_length')
 
     with self.argument_context('containerapp env dapr-component') as c:
-        c.argument('dapr_app_id', help="The Dapr app ID.")
-        c.argument('dapr_app_port', help="The port of your app.")
-        c.argument('dapr_app_protocol', help="Tell Dapr which protocol your application is using.  Allowed values: grpc, http.")
```

Comment:
May I ask if these parameters are not available anymore after this change? will this change make a breaking change for customers?

Reply:
These parameters are not supported before, they are unused.
So it will not be a breaking change.